lib/lualinux.c

```diff
@@ -104,6 +105,34 @@ static int lualinux_lookup(lua_State *L)
 	return 1;
 }
 
+#define LUALINUX_NEW_ENDIAN_CONVERTER(method, T) \
```

Comment:
```suggestion
#define LUALINUX_NEW_BYTESWAPPER(func, T) \
```

"byte swapper" is a bit shorter than "endian converter".. and `method` is a bit misleading as it is not operating on an object..

Reply:
good work on this template, btw 💪